include/ros1_bridge/factory.hpp

```diff
@@ -152,6 +152,92 @@ class Factory : public FactoryInterface
     ROS1_T & ros1_msg);
 };
 
+template <class R1_TYPE, class R1_REQ, class R1_RES, class R2_TYPE, class R2_REQ, class R2_RES>
```

Comment:
Please remove the template arguments for request / response. They can be derived with e.g. `R1_TYPE::Request`.